src/main/java/com/it4045/pokemonviewer/enterprise/dto/Pokemon.java

```diff
@@ -3,51 +3,52 @@
 import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
+import java.lang.reflect.Type;
 import java.util.List;
 
 public @Data
 class Pokemon {
 
     @SerializedName("id")
-    private int pokemonId;
+    private int id;
     @SerializedName("num")
-    private String pokemonNumber;
+    private int number;
     @SerializedName("name")
-    private String pokemonName;
+    private String name;
     @SerializedName("img")
-    private String pokemonImg;
+    private String img;
     @SerializedName("type")
-    private List<String> pokemonType;
+    private List<String> type;
     @SerializedName("height")
-    private String pokemonHeight;
+    private double height;
     @SerializedName("weight")
-    private String pokemonWeight;
+    private double weight;
```

Comment:
weight and height should be double because they cannot be string until written in  words.